### DISCUSSION TITLE
part-ii/2014-98/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#78] [Question comments]
Part II 2014 Number Theory: Paper 4, Section I, F 

https://questions.tripos.org/part-ii/2014-98/

**Comment by walrus1729** [comment]
<details>
<summary>Solution</summary>

Theorem: (Chinese Remainder Theorem) Let $m_1, \dots, m_n$ be pairwise coprime integers and let $a_1, \dots, a_n \in \mathbb{Z}$. Then the system of congruences

$$x \equiv a_i \pmod{m_i}, 1 \leq i \leq n$$

is soluble and the solution is unique $\mod M$, where $M = \prod_{i=1}^{n} m_i$.

By the Chinese Remainder Theorem, this system of congruences will have a unique solution $\mod 105$ (since $105 = 3 \cdot 5 \cdot 7$). We solve the first two congruences by inspection to get that $x \equiv 8 \pmod{15}$. For the third congruence, we can use inspection as well. Alternatively, we can use Euclid's algorithm to get that $1 = 15 - 2 \cdot 7$, so $-2 \cdot 7 = -14 \equiv 1 \pmod{15}$ and $-14 \equiv 0 \pmod{7}$, and $15 \equiv 1 \pmod{7}$ and $15 \equiv 0 \pmod{15}$. Hence we have that

$$x \equiv 5 \cdot 15 + 8 \cdot (-14) \pmod{105}$$

so

$$x \equiv - 37 \equiv 68 \pmod{105}$$

For the last part, we apply the Chinese Remainder Theorem again. For $1 \leq i \leq 100$, let $p_i$ be the $i$th prime, and consider the system of simultaneous congruences

$$x \equiv -i \pmod{{p_i}^2}, 1 \leq i \leq 100$$

Since the ${p_i}^2$ are pairwise coprime, this system has a solution by the Chinese Remainder Theorem, and for $1 \leq i \leq 100$, $x + i$ is divisible by ${p_i}^2$ and is therefore not squarefree. Therefore $x+1,x+2,\dots,x+100$ are 100 consecutive numbers, none of which are squarefree.

</details>
